### PR TITLE
8 add project page

### LINKED ISSUE #8: Add: project page
Is your feature request related to a problem? Please describe.
N/A

Describe how the solution should behave
The single project page should display the details and allow the user to apply to a project. It should be in the `pages` directory and have 2 components as the content and the applicant list and they should be in the `components` directory. The applicant list will contain a table and it is corresponding components, and the content will be the project detail and the other data.

Screenshots of the design/if any
image

Additional context
N/A

### PR DESCRIPTION
**Pull request by ZaminMirzad**
- Project page added and designed + responsive
- ProjectContent component added
- ApplicantList compnent added
- navbar and footer pulled from their branches for easing the work
- some minor bug fixes within navbar and footer components

image
image
image
image
image

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -1,2 +1,7 @@
-export const HOME_ROUTE = 'home';
-export const CONTACT_ROUTE = 'contact';
+export const HOME_ROUTE = '/';
+export const CONTACT_ROUTE = 'contact';
+export const ABOUT_ROUTE = 'about';
+export const LOGIN_ROUTE = 'login';
+export const SIGN_UP_ROUTE = 'signup';
+export const PROJECT_ROUTE = 'project';
```
Comment (MuhannedNoman) [inline src/routes.js]:
```suggestion
export const CONTACT_ROUTE = '/contact';
export const ABOUT_ROUTE = '/about';
export const LOGIN_ROUTE = '/login';
export const SIGN_UP_ROUTE = '/signup';
export const PROJECT_ROUTE = '/project';

```

File: src/components/ProjectContent.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import projectimg from '../assets/project.jpg';
+
+function ProjectContent() {
+  return (
+    <div className="grid grid-cols-1 sm:grid-cols-1 md:grid-cols-2 gap-8  p-3">
+      <div className="flex flex-col p-3 ">
+        <img
+          src={projectimg}
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: Let's make this a prop that gets passed into this component.

File: src/components/ProjectContent.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  return (
+    <div className="grid grid-cols-1 sm:grid-cols-1 md:grid-cols-2 gap-8  p-3">
+      <div className="flex flex-col p-3 ">
+        <img
+          src={projectimg}
+          alt="project"
+          className="sm:h-48 sm:w-auto md:w-auto lg:w-80  rounded"
+        />
+        <h1 className="text-center sm:text-center md:text-left lg:text-left text-2xl text-blue-500 py-3 font-normal">
+          Project Name
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: Should be a prop as well.

File: src/components/ProjectContent.jsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        </h1>
+        <span className="text-sm text-center sm:text-center md:text-left lg:text-left">
+          Mar 12 2022
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: Props as well.

File: src/components/ProjectContent.jsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+        </span>
+        <span className="text-center sm:text-center md:text-left lg:text-left pb-4">
+          Part-Time
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: Prop

File: src/components/ProjectContent.jsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        <p className="flex flex-col items-center lg:flex md:flex">
+          You can load project details such as the project name, classification,
+          and type, as well as the start and end dates, to existing and new
+          projects. You load project details to existing projects by using a
+          data file with the same format as the ProjectDetails.csv data load
+          file
+          <br />
+          <br />
+          You load project details to existing projects by using a data file
+          with the same format as the ProjectDetails.csv data load file
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: The details of the project should be a prop as well.

File: src/components/ProjectContent.jsx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+            >
+              Apply
+            </button>
+          </div>
+        </p>
+      </div>
+      <div className="md:flex md:flex-col md:justify-center p-3 md:pt-28 ">
+        <p>
+          <span className="text-blue-600 font-bold">Skills: </span>
+          Public speaking, Teaching, Good communication...
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: Props

File: src/components/ProjectContent.jsx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+        </p>
+        <p>
+          <span className="text-blue-600 font-bold">Address: </span>
+          16 Woodlawn Terrace, Waterbury, ct, 6310 United States
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: Prop

File: src/components/ProjectContent.jsx [hunk 9]
```diff
@@ -0,0 +1,60 @@
+import projectimg from '../assets/project.jpg';
+
+function ProjectContent() {
```
Comment (MuhannedNoman) [inline src/components/ProjectContent.jsx]: This component should receive all the information that it displays as props.

File: src/components/ApplicantList.jsx [hunk 10]
```diff
@@ -0,0 +1,101 @@
+function ApplicantList() {
```
Comment (MuhannedNoman) [inline src/components/ApplicantList.jsx]: This component should receive an array of applicants as a prop, and render accordingly.

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
Hey @ZaminMirzad again, amazing work as usual. 
Left some notes to make things better.

File: src/components/ApplicantList.jsx (hunk 10 above)
Comment (zaminmirzad) [reply src/components/ApplicantList.jsx]: done

File: src/components/ProjectContent.jsx (hunk 9 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/components/ProjectContent.jsx (hunk 2 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/components/ProjectContent.jsx (hunk 3 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/components/ProjectContent.jsx (hunk 4 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/components/ProjectContent.jsx (hunk 5 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/components/ProjectContent.jsx (hunk 6 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/components/ProjectContent.jsx (hunk 7 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/components/ProjectContent.jsx (hunk 8 above)
Comment (zaminmirzad) [reply src/components/ProjectContent.jsx]: done

File: src/routes.js (hunk 1 above)
Comment (zaminmirzad) [reply src/routes.js]: done

**Review by MuhannedNoman — DISMISSED** [review]

**Comment by MuhannedNoman** [comment]
Hey @ZaminMirzad can you pull this locally and run `npm install` to update package-lock.json. Then push the changes, thank you.

**Comment by ZaminMirzad** [comment]
Ok doing it now

**Pull request merged**